LICENSE.md

```diff
@@ -1,66 +1,117 @@
-11.10. Force Majeure. Neither party will be liable to the other for delays or failures in performance resulting from causes beyond the reasonable control of that party, including, but not limited to, acts of God, labor disputes or disturbances, material shortages or rationing, riots, acts of war, governmental regulations, communications or utility failures, or casualties.
-11.11. Entire Agreement; Amendment. This Agreement, including all Exhibits, reflects the entire agreement of the parties regarding the subject matter hereof, and supersedes all prior or contemporaneous understanding or agreements between the parties, whether written or oral. This Agreement will not be amended, altered or changed except by written agreement signed by both parties that identifies itself as an amendment to this Agreement.
-11.12. Construction of Agreement. This Agreement has been negotiated by the respective parties hereto and their attorneys and the language hereof shall not be construed for or against any party. The titles and headings herein are for reference purposes only and shall not in any manner limit the construction of this Agreement, which shall be considered as a whole. The word "including" is not limiting and will be deemed to be followed by the wording "without limitation."
-11.13. Controlling Language. This Agreement is prepared an executed in the English language only, which language shall be controlling in all respects. Any translations of this Agreement into any other language are for reference only and shall have no legal or other effect. Any notice which is required or permitted to be given by one party to the other under this Agreement will be in the English language and will be in writing. All proceedings related to this Agreement shall be conducted in the English language.
\ No newline at end of file
+# BigDesign UI Kit
+Thank you for your interest in BigDesign.  BigDesign is an open source project subject to the below open source licenses for the different UI components of BigDesign. **BY DOWNLOADING, INSTALLING, OR OTHERWISE USING THE BIGDESIGN PROJECT YOU AGREE TO ABIDE BY THIS LICENSE**. IF YOU DO NOT AGREE WITH THE TERMS AND CONDITIONS OF THIS LICENSE YOU ARE NOT AUTHORIZED TO USE BIGDESIGN.
+
+### Disclaimer
+BigCommerce, Inc. is and remains the owner of all right, title, and interest in its e-commerce platform and other products and services, including all rights in the BigCommerce® mark and other trademarks and logos of BigCommerce. Further, your participation in the BigDesign program may be subject to developer terms, API terms, or other BigCommerce terms relevant to your design of offerings that interface with the BigCommerce platform. BigCommerce reserves all rights not expressly granted hereunder.
+
+## BigDesign Software Code
+**Applies to**: All CSS and JS files in at https://github.com/bigcommerce/big-design.
```

Comment:
This is more of a note, than it is a comment to change. But currently we don't have any CSS files, I'm guessing this is for future outlook or in-case we add one?

Reply:
I'm guessing this list should include `JSX`, `TSX`, `JS`, and `TS` files?